article/_posts/2021-07-28-aavae.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: review
+title: "AAVAE: Augmentation-Augmented Variational Autoencoders"
+tags: autoencoder VAE representation-learning self-supervised
+author: ""
+cite:
+    authors: "William Falcon, Ananya Harsh Jha, Teddy Koker, Kyunghyun Cho"
+    title:   ""
```

Comment:
I pushed a change that allows letting this field blank; the title of the page will be used (since it's pretty much always the same).